src/components/Amount.tsx

```diff
@@ -32,7 +32,7 @@ export default function Amount({
   }
 
   return (
-    <CardBackground className="flex flex-col gap-2 px-6 py-4 md:min-w-[220px] md:gap-4">
```

Comment:
Font size too small, should be 26px but was 22px

<img width="1826" alt="image" src="https://github.com/user-attachments/assets/23c44b22-6436-44ec-a3e0-755c7b8720a6">